common/language.cpp

```diff
@@ -41,26 +63,25 @@ const LanguageDescription g_languages[] = {
 	{    "fi", "fi_FI", "Finnish", FI_FIN },
 	{    "fr", "fr_FR", "French", FR_FRA },
 	{    "de", "de_DE", "German", DE_DEU },
-	{    "gr", "el_GR", "Greek", GR_GRE },
+	{    "el", "el_GR", "Greek", EL_GRC },
 	{    "he", "he_IL", "Hebrew", HE_ISR },
-	{    "hb", "he_IL", "Hebrew", HE_ISR }, // Deprecated
 	{    "hu", "hu_HU", "Hungarian", HU_HUN },
 	{    "it", "it_IT", "Italian", IT_ITA },
-	{    "jp", "ja_JP", "Japanese", JA_JPN },
```

Comment:
I just realized that changing 1st column may break some ConfMan and AdvancedDetector related stuff with games played with previous versions of ScummVM. See `getLanguageCode` function.